geniza/corpus/tests/test_corpus_templates.py

```diff
@@ -136,6 +136,16 @@ def test_editors(self, client, document, source, twoauthor_source):
         response = client.get(document.get_absolute_url())
         assertContains(response, "Editors")
 
+    def test_multiple_shelfmarks(self, client, document, join):
+        """Document detail template should show multiple shelfmarks"""
+        response = client.get(document.get_absolute_url())
+        # TODO: This is fragile, consider something better
+        assertNotContains(response, "Shelfmark", html=True)
```

Comment:
you're not actually using an html match here, but that would be more robust — why not include the full `<dt>` tag?

Reply:
ff40fa023ef5d9811b1622c51b2692d64423e05a ✅